src/commands/server/status.ts

```diff
@@ -0,0 +1,74 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command, flags } from '@oclif/command'
+import { cli } from 'cli-ux'
+import * as notifier from 'node-notifier'
+
+import { KubeHelper } from '../../api/kube'
+import { VersionHelper } from '../../api/version'
+import { cheNamespace } from '../../common-flags'
+
+export default class List extends Command {
+  // Implementation-Version it is a property from Manifest.ml inside of che server pod which indicate Eclipse Che build version.
+  readonly chePreffixVersion = 'Implementation-Version: '
+  readonly cheServerSelector = 'app=che,component=che'
```

Comment:
It is already there https://github.com/che-incubator/chectl/blob/master/src/tasks/che.ts#L33